src/javascripts/components/search.tracking.js

```diff
@@ -0,0 +1,119 @@
+import 'govuk-frontend/vendor/polyfills/Element/prototype/classList'
+import common from 'govuk-frontend/common'
+var nodeListForEach = common.nodeListForEach
+
+function addToDataLayer (payload) {
+  window.dataLayer = window.dataLayer || []
+  window.dataLayer.push(payload)
+}
+
+function stripPossiblePII (string) {
+    // Try to detect emails and redact it.
+  string = string.replace(/\S*@\S*\s?/g, '[blocked]')
+    // If someone has typed in a number it's likely not related so redact it
+  string = string.replace(/0|1|2|3|4|5|6|7|8|9/g, '[blocked]')
+  return string
+}
+
+function getSearchResults ($module) {
+  var $options = $module.querySelectorAll('.app-site-search__option')
+
+  var results = []
+  nodeListForEach($options, function ($option, key) {
+    var $section = $option.querySelector('.app-site-search--section')
```

Comment:
Would be good if autocomplete had events for results that we could use to get the raw data, this code gives me the creeps, but would be the same if we would have done this in Google Tag Manager...

Reply:
Wait could we get this from the original data source hmmm

Reply:
hmm might be able to use the renderResults function